package.json

```diff
@@ -44,11 +44,14 @@
     "http-status-codes": "^2.2.0",
     "launchdarkly-js-client-sdk": "^2.22.1",
     "lodash": "^4.17.21",
+    "pdfjs-dist": "3.4.120",
     "pinia": "^2.0.33",
     "pinia-class": "^0.0.3",
     "regenerator-runtime": "^0.13.9",
     "sbc-common-components": "3.0.5",
+    "tiptap-vuetify": "^2.24.0",
```

Comment:
We'll need to replace with Vue3

Reply:
Is this an item in another ticket?

Reply:
nope

Reply:
Please add a ticket this this epic (similar to other tickets in there).
https://app.zenhub.com/workspaces/entities-team-space-6143567664fb320019b81f39/issues/gh/bcgov/entity/13576

Reply:
New ticket is bcgov/entity#16464.

Reply:
Is this dependency going to be added for Create UI and Filings UI as well?

Reply:
This is an editor, not needed in Filings or Create UI yet...